google-cloud-notebooks/src/test/java/com/google/cloud/notebooks/v1beta1/it/ITNotebookServiceClientTest.java

```diff
@@ -0,0 +1,233 @@
+        assertEquals(expectedNotebookInstance.getContainerImage(), element.getContainerImage());
+        assertEquals(expectedNotebookInstance.getName(), element.getName());
+        assertEquals(expectedNotebookInstance.getMachineType(), element.getMachineType());
+        assertEquals(expectedNotebookInstance.getNetwork(), element.getNetwork());
+        assertEquals(expectedNotebookInstance.getSubnet(), element.getSubnet());
+      }
+    }
+  }
+
+  @Test
+  public void testGetEnvironment() {
+    GetEnvironmentRequest environmentRequest =
+        GetEnvironmentRequest.newBuilder().setName(ENVIRONMENT_NAME).build();
+    Environment expectedEnvironmentResponse = client.getEnvironment(environmentRequest);
+    assertEquals(expectedEnvironmentResponse.getName(), expectedEnvironmentResponse.getName());
+    assertEquals(
+        expectedEnvironmentResponse.getContainerImage(),
+        expectedEnvironmentResponse.getContainerImage());
+  }
+
+  @Test
+  public void testListEnvironment() {
+    ListEnvironmentsRequest request =
+        ListEnvironmentsRequest.newBuilder().setParent(PARENT).build();
+    for (Environment element : client.listEnvironments(request).iterateAll()) {
+      if (element.getName().equals(ENVIRONMENT_ID)) {
+        assertEquals(expectedEnvironmentResponse.getName(), element.getName());
+        assertEquals(expectedEnvironmentResponse.getContainerImage(), element.getContainerImage());
+      }
+    }
+  }
+
+  @Test
+  public void testSetInstanceLabels() throws ExecutionException, InterruptedException {
+    SetInstanceLabelsRequest setInstanceLabelsRequest =
+        SetInstanceLabelsRequest.newBuilder().setName(INSTANCE_NAME).putLabels("a", "100").build();
+    Instance response = client.setInstanceLabelsAsync(setInstanceLabelsRequest).get();
+    assertEquals(response.getLabelsMap().get("a"), "100");
+  }
+
+  @Test
+  public void testSetInstanceMachineType() throws ExecutionException, InterruptedException {
+    this.stopInstance();
+    SetInstanceMachineTypeRequest request =
+        SetInstanceMachineTypeRequest.newBuilder()
+            .setName(INSTANCE_NAME)
+            .setMachineType(MACHINE_TYPE_B)
+            .build();
+    Instance response = client.setInstanceMachineTypeAsync(request).get();
+    assertTrue(response.getMachineType().endsWith(MACHINE_TYPE_B));
+    this.startInstance();
+  }
+
+  @Test
+  public void testStartInstance() throws ExecutionException, InterruptedException {
+    String state = this.stopInstance();
+    state = this.startInstance();
+    assertEquals(state, "PROVISIONING");
+  }
+
+  @Test
+  public void testStopInstance() throws ExecutionException, InterruptedException {
+    String state = this.stopInstance();
+    assertEquals(state, "STOPPED");
+    state = this.startInstance();
+  }
+
+  @Test
+  public void testSetInstanceAccelarator() throws ExecutionException, InterruptedException {
+    Instance.AcceleratorType type = Instance.AcceleratorType.NVIDIA_TESLA_P4;
```

Comment:
remove this `Instance.AcceleratorType type = Instance.AcceleratorType.NVIDIA_TESLA_P4;`

Reply:
Removed